modules/Bio/Otter/Auth/SSO.pm

```diff
@@ -55,13 +55,13 @@ sub login {
 
     # need to url-encode these
     my $user  = uri_escape($orig_user); # possibly not worth it...
-    $password = uri_escape($password);  # definitely worth it!
+
```

Comment:
Why do we skip escaping? It definitely will  cause troubles.
Characters that are not allowed should be forbidden in user interface, without skipping safety escaping.

Reply:
I forgot why I removed it. Will be included.